src/com/seafile/seadroid2/SeafConnection.java

```diff
@@ -888,6 +888,35 @@ public String getShareLink(String repoID, String path, boolean isdir) throws Sea
         }
     }
 
+    public Pair<String, String> star(String repoID, String path) throws SeafException {
+        try {
+            HttpRequest req = prepareApiPostRequest("api2/starredfiles/", true, null);
+
+            req.form("repo_id", repoID);
+            req.form("p", path);
+
+            checkRequestResponseStatus(req, HttpURLConnection.HTTP_OK);
```

Comment:
The status code of a successful file start request is `201`, so here we should use `HTTP_CREATED` instead of `HTTP_OK`.